cmd/agent/flags.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+)
+
+var flagSendAddr string
+var reportInterval int
+var pollInterval int
+
+func parseFlags() {
+	flag.StringVar(&flagSendAddr, "a", "localhost:8080", "address and port to run server")
+	flag.IntVar(&reportInterval, "r", 10, "reportInterval description")
+	flag.IntVar(&pollInterval, "p", 2, "pollInterval description")
+	flag.Parse()
+
+	if flag.NArg() > 0 {
+		fmt.Println("Unknown flag:", flag.Arg(0))
+		flag.PrintDefaults()
+		os.Exit(1)
```

Comment:
It would be better if you don't use `os.Exit` here.

> os.Exit skips the execution of deferred function.

It might be dangerous. So, please return an error.

Reply:
++